src/clientsettingsdlg.cpp

```diff
@@ -396,6 +396,13 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
 
     UpdateSoundCardFrame();
 
+    // add "Delete Server" to cbxCentralServerAddress context menu
+    cbxCentralServerAddress->setContextMenuPolicy ( Qt::CustomContextMenu ) ;
+    m_deleteAction = new QAction ( tr ( "Delete Server" ), this );
+    connect ( m_deleteAction, SIGNAL ( triggered() ),
```

Comment:
See lines 407 onward.  Could you move your connections to that section and follow the same code style, please.  (I know it's Qt4 style...)

Reply:
Done, also in connectdlg, any other comments?
I'll push all at once.